website/content/en/pre-docs/AWS/constraints.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Provisioning Configuration"
+linkTitle: "Provisioning"
+weight: 10
+---
+
+The [Provisioner CRD]({{< ref "../reference/provisioner-crd.md" >}}) provides two sections for constraining nodes. 
+
+- [`spec.requirements`](../reference/provisioner-crd#specrequirements)

```

Comment:
Does it make sense to link to the NodeSelection section of the Tasks / Running Pods doc?